server/routes/post.router 2.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+
+/**
+ * GET route template
+ */
+router.get('/', (req, res) => {
+  // GET route code here
+});
```

Comment:
This file is called `post.router 2.js`? It has a `GET` in it and spaces in file names are not so good.